src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -186,7 +186,6 @@ public void init() {
         }
         else {
             final int[] acceptableJavadocTokens = getAcceptableJavadocTokens();
-            Arrays.sort(acceptableJavadocTokens);
```

Comment:
this line here for a reason. it is necessary to make sure that tokens are sorted to perform binary search in if condition below

Reply:
Yes, but it is assured that it is sorted. It can only be unsorted in a single case when `getRequiredJavadocTokens()` is null which can practically never be null.

Reply:
To make a test case to kill this mutation, you need to make the binary search fail.

Reply:
I would probably just do something like https://github.com/checkstyle/checkstyle/blob/1e3fb3fae940bf7ea340592f7f0cda3af1b320e3/src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java#L625 and make a "fake" check with lots of tokens.

Reply:
Thanks, it definitely makes the job easy.

Reply:
> Thanks, it definitely makes the job easy.

Only if it works :)

Reply:
Done, well that didn't work out. Found out a better way. :)